data/logement/vacances.yaml

```diff
@@ -0,0 +1,292 @@
+logement . vacances:
+  titre: Nuitées week-end ou vacances
+  mosaique:
+    type: selection
+    clé: présent
+  question: Comment êtes-vous hébergé pour vos week-ends, vos vacances ?
+  icônes: 🏖
+  description: |
+    Renseignez ici les différents types de logement que vous occupez pour vos voyages (motif personnel).
+
+    > Pour le moment, nous ne proposons pas la maison secondaire et les locations type Airbnb, on espère vous proposez très prochainement les autres modes d'hébergement !
+  note: |
+    ### Qu'entend-on par "nuitées hors logement" dans Nos Gestes Climat?
+
+      - Hôtel ou chambre privée
+      - Emplacement en camping
+      - Auberge de jeunesse
+      - Locations de vacances (type Airbnb)
+      - Famille ou amis
+      - Maison secondaire
+
+    Sont donc exclues les nuits en hôtel pour **raisons professionnelles**.
+
+    ### Vers une première version
+
+    Dans un premier temps, l'idée est de restreindre le périmètre de la V1 au secteur de l'hotellerie.
+
+    En effet :
+
+    - Nous choisissons d'exclure les airbnb qui se rapprochent davantage d'une méthode de comptabilité liée à la maison secondaire. 
+    On pourrait effectivement demander si un propriétaire loue sa maison secondaire : c'est à ce moment-là que nous aurons à la fois 
+    la méthode de calcul mais également de l'attribution des résultats entre loueurs et propriétaire.
+    - Il est difficile de trouver des données fiables sur l'empreinte des nuitées en camping : on pourrait imaginer que l'impact 
+    est inférieur à une nuit d'hôtel. Peut-être qu'une manière de prendre en compte l'empreinte d'un emplacement serait de prendre une partie seulement de l'empreinte d'une nuit en hôtel en excluant par exemple l'empreinte de la blanchisserie.
+    - Pour l'empreinte d'une nuit au sein de sa famille ou chez des amis, l'empreinte est aujourd'hui indirectement comptée dans l'empreinte des accueillants.
+
+    En ce qui concerne les hôtels, les auberges de jeunesse, les "resorts", nous considérons un facteur d'émission commun pour le moment.
+
+    Dans cette première version, l'empreinte de l'hôtel sera divisée par le nombre d'habitants du logement.
```

Comment:
L'hypothèse me semble acceptable pour éviter de rajouter une question sur le nombres de personnes partageant la chambre. 
Mais ne pourrait-on pas avoir un "garde fous" type si nb_hab > 5 alors nb_nuitées * 2 car pour des familles de plus de 5 (voire 4 même) il me parait évident qu'on a besoin de deux chambres (ce qui revient en quelque sorte à doublée le nombre de nuitées). Qu'en pensez-vous @Clemog @JuliePouliquen

Reply:
Effectivement c'est une bonne idée ! Tu veux dire que une famille de plus de 4 personnes prendra forcément 2 chambres ? Je peux prendre la modif

Après l'hypothèse à la base présente des limites .. Si finalement ce ne sont que des we entre parents par exemple

Reply:
C'est fait !